### DISCUSSION TITLE
Coverlet is generating 0% coverage

### DISCUSSION THREAD

**Discussion by Oblutack** [coverlet-coverage/coverlet#1556] [Q&A]
I test with dotnet test using msbuild v6.0.0. It doesn't generate anything. I tried using collector and running dotnet test --collect:"XPlat Code Coverage" but that only shows my passed tests and nothing else no %.
https://github.com/Oblutack/TS_Zadaca2

**Comment by Bertk** [comment]
Sorry for the late reply. The latest released coverlet version is meanwhile 6.0.4.
coverlet has some requirements for SDK and runtime. Please check documentation.

I guess your problem is already solved and will close this discussion.
